medcat/config.py

```diff
@@ -228,6 +228,15 @@ def __init__(self) -> None:
                 }
 
         self.linking: Dict[str, Any] = {
+                # Checkpointing config
+                'checkpoint': {
+                    # When doing training this is the name of the directory where checkpoints will be saved
+                    'output_dir': 'checkpoints',
+                    # When training how often to save the checkpoint (one step == one document)
+                    'steps': 1000,
```

Comment:
I would disable ckpts by default, but could be just my preference - not sure what is standard.

Reply:
Set the default steps to 100K which would "disable" the checkpoint saving for most cases. Users can set its value to max int if needed.